sources/whatsnew.asciidoc

```diff
@@ -10,8 +10,8 @@
 [[whatsnew-20]]
 === What's new in 2.0
 
-The main contribution of {spec-name-bv} 2.0 is leveraging the new language features and API additions of Java 8 for the purposes of validation.
-Java 8 or later is required to use {spec-name-bv} 2.0.
+The main contribution of Bean Validation 2.0 is leveraging the new language features and API additions of Java 8 for the purposes of validation.
+Java 8 or later is required to use Bean Validation 2.0.
```

Comment:
I don't think it's a good idea to mention Jakarta in the changelog, in the context of BV 2.0 and BV 1.1.

I could envision that we do it for 2.0 (even if it's rewriting history) but clearly not for 1.1.

Reply:
OK as I am not sure, I've created a  separate commit (like yours) but added a note explaining why we don't change,